README.md

```diff
@@ -98,6 +98,8 @@ Beautifier Options:
   -k, --keep-array-indentation      Preserve array indentation
   -x, --unescape-strings            Decode printable characters encoded in xNN notation
   -w, --wrap-line-length            Wrap lines at next opportunity after N characters [0]
+  -a, --wrap-attributes             Wrap attributes to new lines
```

Comment:
I'll move them down and change the -a to -W, I think it makes more sense than -A if it's colliding with other options since it is related to wrap (-w). Also looking into the options.